scalafix/input/src/main/scala/fix/Collectionstrawman_v0_BitSet.scala

```diff
@@ -0,0 +1,11 @@
+/*
+rule = "scala:fix.Collectionstrawman_v0"
+ */
+package fix
+
+import scala.collection.{BitSet, immutable, mutable}
+object Collectionstrawman_v0_BitSet {
+  val xs: BitSet = ???
```

Comment:
`scala.collection.BitSet` has a companion object but `strawman.collection.BitSet` doesn't. Is it intentional or missing?

Reply:
That’s an oversight, thanks for reporting :)